core/conversion/converters/impl/constant_pad.cpp

```diff
@@ -0,0 +1,156 @@
+#include <ATen/ATen.h>
+#include <vector>
+#include "NvInfer.h"
+#include "core/conversion/converters/converters.h"
+#include "core/util/prelude.h"
+#include "torch/torch.h"
+
+namespace trtorch {
+namespace core {
+namespace conversion {
+namespace converters {
+namespace impl {
+namespace {
+
+auto replication_pad_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().pattern(
+    {"aten::constant_pad_nd(Tensor self, int[] pad, Scalar value=0) -> (Tensor)",
+     [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+       auto in = args[0].ITensor();
+       auto inDims = in->getDimensions();
+       int64_t inRank = inDims.nbDims;
+       auto padding = args[1].unwrapToIntList().vec();
+       int64_t padSize = padding.size();
+       auto value = args[2].unwrapToScalar().to<float>();
+
+       TRTORCH_CHECK(padSize % 2 == 0, "Length of pad must be even but instead it equals " << padSize);
+
+       int64_t l_pad = padSize / 2;
+       TRTORCH_CHECK(
+           inRank >= (int64_t)l_pad,
+           "Length of pad should be no more than twice the number of "
+           "dimensions of the input. Pad length is "
+               << padSize << "while the input has " << inRank << "dimensions.");
+
+       // TODO negative padding
```

Comment:
Could we file an issue on this if its something that needs to get done just so we don't forget?

Reply:
> Could we file an issue on this if its something that needs to get done just so we don't forget?

Yeah，I have filed an issue about the TODO. #482